## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 492830`
Commit: `be5df37000433d22d21ba6f69f63ab1c6a946306`

---
### `x86_64-linux`
<details>
  <summary>:x: 93 packages failed to build:</summary>
  <ul>
    <li>expliot</li>
    <li>expliot.dist</li>
    <li>python313Packages.angrcli</li>
    <li>python313Packages.angrcli.dist</li>
    <li>python314Packages.angrcli</li>
    <li>python314Packages.angrcli.dist</li>
    <li>python314Packages.cliff</li>
    <li>python314Packages.cliff.dist</li>
    <li>python314Packages.hacking</li>
    <li>python314Packages.hacking.dist</li>
    <li>python314Packages.jenkins-job-builder</li>
    <li>python314Packages.jenkins-job-builder.dist</li>
    <li>python314Packages.keystoneauth1</li>
    <li>python314Packages.keystoneauth1.dist</li>
    <li>python314Packages.openstacksdk</li>
    <li>python314Packages.openstacksdk.dist</li>
    <li>python314Packages.openstacksdk.man</li>
    <li>python314Packages.os-client-config</li>
    <li>python314Packages.os-client-config.dist</li>
    <li>python314Packages.osc-lib</li>
    <li>python314Packages.osc-lib.dist</li>
    <li>python314Packages.osc-placement</li>
    <li>python314Packages.osc-placement.dist</li>
    <li>python314Packages.oslo-concurrency</li>
    <li>python314Packages.oslo-concurrency.dist</li>
    <li>python314Packages.oslo-config</li>
    <li>python314Packages.oslo-config.dist</li>
    <li>python314Packages.oslo-context</li>
    <li>python314Packages.oslo-context.dist</li>
    <li>python314Packages.oslo-db</li>
    <li>python314Packages.oslo-db.dist</li>
    <li>python314Packages.oslo-i18n</li>
    <li>python314Packages.oslo-i18n.dist</li>
    <li>python314Packages.oslo-log</li>
    <li>python314Packages.oslo-log.dist</li>
    <li>python314Packages.oslo-metrics</li>
    <li>python314Packages.oslo-metrics.dist</li>
    <li>python314Packages.oslo-serialization</li>
    <li>python314Packages.oslo-serialization.dist</li>
    <li>python314Packages.oslo-utils</li>
    <li>python314Packages.oslo-utils.dist</li>
    <li>python314Packages.osprofiler</li>
    <li>python314Packages.osprofiler.dist</li>
    <li>python314Packages.pyghmi</li>
    <li>python314Packages.pyghmi.dist</li>
    <li>python314Packages.python-aodhclient</li>
    <li>python314Packages.python-aodhclient.dist</li>
    <li>python314Packages.python-barbicanclient</li>
    <li>python314Packages.python-barbicanclient.dist</li>
    <li>python314Packages.python-cinderclient</li>
    <li>python314Packages.python-cinderclient.dist</li>
    <li>python314Packages.python-designateclient</li>
    <li>python314Packages.python-designateclient.dist</li>
    <li>python314Packages.python-glanceclient</li>
    <li>python314Packages.python-glanceclient.dist</li>
    <li>python314Packages.python-heatclient</li>
    <li>python314Packages.python-heatclient.dist</li>
    <li>python314Packages.python-ironicclient</li>
    <li>python314Packages.python-ironicclient.dist</li>
    <li>python314Packages.python-jenkins</li>
    <li>python314Packages.python-jenkins.dist</li>
    <li>python314Packages.python-keystoneclient</li>
    <li>python314Packages.python-keystoneclient.dist</li>
    <li>python314Packages.python-magnumclient</li>
    <li>python314Packages.python-magnumclient.dist</li>
    <li>python314Packages.python-manilaclient</li>
    <li>python314Packages.python-manilaclient.dist</li>
    <li>python314Packages.python-mistralclient</li>
    <li>python314Packages.python-mistralclient.dist</li>
    <li>python314Packages.python-neutronclient</li>
    <li>python314Packages.python-neutronclient.dist</li>
    <li>python314Packages.python-novaclient</li>
    <li>python314Packages.python-novaclient.dist</li>
    <li>python314Packages.python-octaviaclient</li>
    <li>python314Packages.python-octaviaclient.dist</li>
    <li>python314Packages.python-openstackclient</li>
    <li>python314Packages.python-openstackclient.dist</li>
    <li>python314Packages.python-swiftclient</li>
    <li>python314Packages.python-swiftclient.dist</li>
    <li>python314Packages.python-troveclient</li>
    <li>python314Packages.python-troveclient.dist</li>
    <li>python314Packages.python-watcherclient</li>
    <li>python314Packages.python-watcherclient.dist</li>
    <li>python314Packages.python-zaqarclient</li>
    <li>python314Packages.python-zaqarclient.dist</li>
    <li>python314Packages.python-zunclient</li>
    <li>python314Packages.python-zunclient.dist</li>
    <li>python314Packages.stestr</li>
    <li>python314Packages.stestr.dist</li>
    <li>python314Packages.tempest</li>
    <li>python314Packages.tempest.dist</li>
    <li>sccmhunter</li>
    <li>sccmhunter.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 112 packages built:</summary>
  <ul>
    <li>baboossh</li>
    <li>baboossh.dist</li>
    <li>barbicanclient (python313Packages.python-barbicanclient)</li>
    <li>barbicanclient.dist (python313Packages.python-barbicanclient.dist)</li>
    <li>bashate</li>
    <li>bashate.dist</li>
    <li>faraday-cli</li>
    <li>faraday-cli.dist</li>
    <li>glanceclient (python313Packages.python-glanceclient)</li>
    <li>glanceclient.dist (python313Packages.python-glanceclient.dist)</li>
    <li>heatclient (python313Packages.python-heatclient)</li>
    <li>heatclient.dist (python313Packages.python-heatclient.dist)</li>
    <li>html5validator</li>
    <li>html5validator.dist</li>
    <li>ironicclient (python313Packages.python-ironicclient)</li>
    <li>ironicclient.dist (python313Packages.python-ironicclient.dist)</li>
    <li>jenkins-job-builder (python313Packages.jenkins-job-builder)</li>
    <li>jenkins-job-builder.dist (python313Packages.jenkins-job-builder.dist)</li>
    <li>kolla</li>
    <li>kolla.dist</li>
    <li>magnumclient (python313Packages.python-magnumclient)</li>
    <li>magnumclient.dist (python313Packages.python-magnumclient.dist)</li>
    <li>manilaclient (python313Packages.python-manilaclient)</li>
    <li>manilaclient.dist (python313Packages.python-manilaclient.dist)</li>
    <li>mistralclient (python313Packages.python-mistralclient)</li>
    <li>mistralclient.dist (python313Packages.python-mistralclient.dist)</li>
    <li>neutronclient (python313Packages.python-neutronclient)</li>
    <li>neutronclient.dist (python313Packages.python-neutronclient.dist)</li>
    <li>nmap-parse</li>
    <li>openstackclient (python313Packages.python-openstackclient)</li>
    <li>openstackclient-full</li>
    <li>openstackclient-full.dist</li>
    <li>openstackclient.dist (python313Packages.python-openstackclient.dist)</li>
    <li>python313Packages.cliff</li>
    <li>python313Packages.cliff.dist</li>
    <li>python313Packages.cmd2</li>
    <li>python313Packages.cmd2-ext-test</li>
    <li>python313Packages.cmd2-ext-test.dist</li>
    <li>python313Packages.cmd2.dist</li>
    <li>python313Packages.hacking</li>
    <li>python313Packages.hacking.dist</li>
    <li>python313Packages.keystoneauth1</li>
    <li>python313Packages.keystoneauth1.dist</li>
    <li>python313Packages.openstacksdk</li>
    <li>python313Packages.openstacksdk.dist</li>
    <li>python313Packages.openstacksdk.man</li>
    <li>python313Packages.os-client-config</li>
    <li>python313Packages.os-client-config.dist</li>
    <li>python313Packages.osc-lib</li>
    <li>python313Packages.osc-lib.dist</li>
    <li>python313Packages.osc-placement</li>
    <li>python313Packages.osc-placement.dist</li>
    <li>python313Packages.oslo-concurrency</li>
    <li>python313Packages.oslo-concurrency.dist</li>
    <li>python313Packages.oslo-config</li>
    <li>python313Packages.oslo-config.dist</li>
    <li>python313Packages.oslo-context</li>
    <li>python313Packages.oslo-context.dist</li>
    <li>python313Packages.oslo-db</li>
    <li>python313Packages.oslo-db.dist</li>
    <li>python313Packages.oslo-i18n</li>
    <li>python313Packages.oslo-i18n.dist</li>
    <li>python313Packages.oslo-log</li>
    <li>python313Packages.oslo-log.dist</li>
    <li>python313Packages.oslo-metrics</li>
    <li>python313Packages.oslo-metrics.dist</li>
    <li>python313Packages.oslo-serialization</li>
    <li>python313Packages.oslo-serialization.dist</li>
    <li>python313Packages.oslo-utils</li>
    <li>python313Packages.oslo-utils.dist</li>
    <li>python313Packages.osprofiler</li>
    <li>python313Packages.osprofiler.dist</li>
    <li>python313Packages.pyghmi</li>
    <li>python313Packages.pyghmi.dist</li>
    <li>python313Packages.pysim</li>
    <li>python313Packages.pysim.dist</li>
    <li>python313Packages.python-aodhclient</li>
    <li>python313Packages.python-aodhclient.dist</li>
    <li>python313Packages.python-cinderclient</li>
    <li>python313Packages.python-cinderclient.dist</li>
    <li>python313Packages.python-designateclient</li>
    <li>python313Packages.python-designateclient.dist</li>
    <li>python313Packages.python-jenkins</li>
    <li>python313Packages.python-jenkins.dist</li>
    <li>python313Packages.python-keystoneclient</li>
    <li>python313Packages.python-keystoneclient.dist</li>
    <li>python313Packages.python-novaclient</li>
    <li>python313Packages.python-novaclient.dist</li>
    <li>python313Packages.python-octaviaclient</li>
    <li>python313Packages.python-octaviaclient.dist</li>
    <li>swiftclient (python313Packages.python-swiftclient)</li>
    <li>swiftclient.dist (python313Packages.python-swiftclient.dist)</li>
    <li>troveclient (python313Packages.python-troveclient)</li>
    <li>troveclient.dist (python313Packages.python-troveclient.dist)</li>
    <li>watcherclient (python313Packages.python-watcherclient)</li>
    <li>watcherclient.dist (python313Packages.python-watcherclient.dist)</li>
    <li>python313Packages.python-zaqarclient</li>
    <li>python313Packages.python-zaqarclient.dist</li>
    <li>zunclient (python313Packages.python-zunclient)</li>
    <li>zunclient.dist (python313Packages.python-zunclient.dist)</li>
    <li>python313Packages.stestr</li>
    <li>python313Packages.stestr.dist</li>
    <li>python313Packages.tempest</li>
    <li>python313Packages.tempest.dist</li>
    <li>python314Packages.cmd2</li>
    <li>python314Packages.cmd2-ext-test</li>
    <li>python314Packages.cmd2-ext-test.dist</li>
    <li>python314Packages.cmd2.dist</li>
    <li>python314Packages.pysim</li>
    <li>python314Packages.pysim.dist</li>
    <li>reno</li>
    <li>reno.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 93 packages failed to build:</summary>
  <ul>
    <li>expliot</li>
    <li>expliot.dist</li>
    <li>python313Packages.angrcli</li>
    <li>python313Packages.angrcli.dist</li>
    <li>python314Packages.angrcli</li>
    <li>python314Packages.angrcli.dist</li>
    <li>python314Packages.cliff</li>
    <li>python314Packages.cliff.dist</li>
    <li>python314Packages.hacking</li>
    <li>python314Packages.hacking.dist</li>
    <li>python314Packages.jenkins-job-builder</li>
    <li>python314Packages.jenkins-job-builder.dist</li>
    <li>python314Packages.keystoneauth1</li>
    <li>python314Packages.keystoneauth1.dist</li>
    <li>python314Packages.openstacksdk</li>
    <li>python314Packages.openstacksdk.dist</li>
    <li>python314Packages.openstacksdk.man</li>
    <li>python314Packages.os-client-config</li>
    <li>python314Packages.os-client-config.dist</li>
    <li>python314Packages.osc-lib</li>
    <li>python314Packages.osc-lib.dist</li>
    <li>python314Packages.osc-placement</li>
    <li>python314Packages.osc-placement.dist</li>
    <li>python314Packages.oslo-concurrency</li>
    <li>python314Packages.oslo-concurrency.dist</li>
    <li>python314Packages.oslo-config</li>
    <li>python314Packages.oslo-config.dist</li>
    <li>python314Packages.oslo-context</li>
    <li>python314Packages.oslo-context.dist</li>
    <li>python314Packages.oslo-db</li>
    <li>python314Packages.oslo-db.dist</li>
    <li>python314Packages.oslo-i18n</li>
    <li>python314Packages.oslo-i18n.dist</li>
    <li>python314Packages.oslo-log</li>
    <li>python314Packages.oslo-log.dist</li>
    <li>python314Packages.oslo-metrics</li>
    <li>python314Packages.oslo-metrics.dist</li>
    <li>python314Packages.oslo-serialization</li>
    <li>python314Packages.oslo-serialization.dist</li>
    <li>python314Packages.oslo-utils</li>
    <li>python314Packages.oslo-utils.dist</li>
    <li>python314Packages.osprofiler</li>
    <li>python314Packages.osprofiler.dist</li>
    <li>python314Packages.pyghmi</li>
    <li>python314Packages.pyghmi.dist</li>
    <li>python314Packages.python-aodhclient</li>
    <li>python314Packages.python-aodhclient.dist</li>
    <li>python314Packages.python-barbicanclient</li>
    <li>python314Packages.python-barbicanclient.dist</li>
    <li>python314Packages.python-cinderclient</li>
    <li>python314Packages.python-cinderclient.dist</li>
    <li>python314Packages.python-designateclient</li>
    <li>python314Packages.python-designateclient.dist</li>
    <li>python314Packages.python-glanceclient</li>
    <li>python314Packages.python-glanceclient.dist</li>
    <li>python314Packages.python-heatclient</li>
    <li>python314Packages.python-heatclient.dist</li>
    <li>python314Packages.python-ironicclient</li>
    <li>python314Packages.python-ironicclient.dist</li>
    <li>python314Packages.python-jenkins</li>
    <li>python314Packages.python-jenkins.dist</li>
    <li>python314Packages.python-keystoneclient</li>
    <li>python314Packages.python-keystoneclient.dist</li>
    <li>python314Packages.python-magnumclient</li>
    <li>python314Packages.python-magnumclient.dist</li>
    <li>python314Packages.python-manilaclient</li>
    <li>python314Packages.python-manilaclient.dist</li>
    <li>python314Packages.python-mistralclient</li>
    <li>python314Packages.python-mistralclient.dist</li>
    <li>python314Packages.python-neutronclient</li>
    <li>python314Packages.python-neutronclient.dist</li>
    <li>python314Packages.python-novaclient</li>
    <li>python314Packages.python-novaclient.dist</li>
    <li>python314Packages.python-octaviaclient</li>
    <li>python314Packages.python-octaviaclient.dist</li>
    <li>python314Packages.python-openstackclient</li>
    <li>python314Packages.python-openstackclient.dist</li>
    <li>python314Packages.python-swiftclient</li>
    <li>python314Packages.python-swiftclient.dist</li>
    <li>python314Packages.python-troveclient</li>
    <li>python314Packages.python-troveclient.dist</li>
    <li>python314Packages.python-watcherclient</li>
    <li>python314Packages.python-watcherclient.dist</li>
    <li>python314Packages.python-zaqarclient</li>
    <li>python314Packages.python-zaqarclient.dist</li>
    <li>python314Packages.python-zunclient</li>
    <li>python314Packages.python-zunclient.dist</li>
    <li>python314Packages.stestr</li>
    <li>python314Packages.stestr.dist</li>
    <li>python314Packages.tempest</li>
    <li>python314Packages.tempest.dist</li>
    <li>sccmhunter</li>
    <li>sccmhunter.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 112 packages built:</summary>
  <ul>
    <li>baboossh</li>
    <li>baboossh.dist</li>
    <li>barbicanclient (python313Packages.python-barbicanclient)</li>
    <li>barbicanclient.dist (python313Packages.python-barbicanclient.dist)</li>
    <li>bashate</li>
    <li>bashate.dist</li>
    <li>faraday-cli</li>
    <li>faraday-cli.dist</li>
    <li>glanceclient (python313Packages.python-glanceclient)</li>
    <li>glanceclient.dist (python313Packages.python-glanceclient.dist)</li>
    <li>heatclient (python313Packages.python-heatclient)</li>
    <li>heatclient.dist (python313Packages.python-heatclient.dist)</li>
    <li>html5validator</li>
    <li>html5validator.dist</li>
    <li>ironicclient (python313Packages.python-ironicclient)</li>
    <li>ironicclient.dist (python313Packages.python-ironicclient.dist)</li>
    <li>jenkins-job-builder (python313Packages.jenkins-job-builder)</li>
    <li>jenkins-job-builder.dist (python313Packages.jenkins-job-builder.dist)</li>
    <li>kolla</li>
    <li>kolla.dist</li>
    <li>magnumclient (python313Packages.python-magnumclient)</li>
    <li>magnumclient.dist (python313Packages.python-magnumclient.dist)</li>
    <li>manilaclient (python313Packages.python-manilaclient)</li>
    <li>manilaclient.dist (python313Packages.python-manilaclient.dist)</li>
    <li>mistralclient (python313Packages.python-mistralclient)</li>
    <li>mistralclient.dist (python313Packages.python-mistralclient.dist)</li>
    <li>neutronclient (python313Packages.python-neutronclient)</li>
    <li>neutronclient.dist (python313Packages.python-neutronclient.dist)</li>
    <li>nmap-parse</li>
    <li>openstackclient (python313Packages.python-openstackclient)</li>
    <li>openstackclient-full</li>
    <li>openstackclient-full.dist</li>
    <li>openstackclient.dist (python313Packages.python-openstackclient.dist)</li>
    <li>python313Packages.cliff</li>
    <li>python313Packages.cliff.dist</li>
    <li>python313Packages.cmd2</li>
    <li>python313Packages.cmd2-ext-test</li>
    <li>python313Packages.cmd2-ext-test.dist</li>
    <li>python313Packages.cmd2.dist</li>
    <li>python313Packages.hacking</li>
    <li>python313Packages.hacking.dist</li>
    <li>python313Packages.keystoneauth1</li>
    <li>python313Packages.keystoneauth1.dist</li>
    <li>python313Packages.openstacksdk</li>
    <li>python313Packages.openstacksdk.dist</li>
    <li>python313Packages.openstacksdk.man</li>
    <li>python313Packages.os-client-config</li>
    <li>python313Packages.os-client-config.dist</li>
    <li>python313Packages.osc-lib</li>
    <li>python313Packages.osc-lib.dist</li>
    <li>python313Packages.osc-placement</li>
    <li>python313Packages.osc-placement.dist</li>
    <li>python313Packages.oslo-concurrency</li>
    <li>python313Packages.oslo-concurrency.dist</li>
    <li>python313Packages.oslo-config</li>
    <li>python313Packages.oslo-config.dist</li>
    <li>python313Packages.oslo-context</li>
    <li>python313Packages.oslo-context.dist</li>
    <li>python313Packages.oslo-db</li>
    <li>python313Packages.oslo-db.dist</li>
    <li>python313Packages.oslo-i18n</li>
    <li>python313Packages.oslo-i18n.dist</li>
    <li>python313Packages.oslo-log</li>
    <li>python313Packages.oslo-log.dist</li>
    <li>python313Packages.oslo-metrics</li>
    <li>python313Packages.oslo-metrics.dist</li>
    <li>python313Packages.oslo-serialization</li>
    <li>python313Packages.oslo-serialization.dist</li>
    <li>python313Packages.oslo-utils</li>
    <li>python313Packages.oslo-utils.dist</li>
    <li>python313Packages.osprofiler</li>
    <li>python313Packages.osprofiler.dist</li>
    <li>python313Packages.pyghmi</li>
    <li>python313Packages.pyghmi.dist</li>
    <li>python313Packages.pysim</li>
    <li>python313Packages.pysim.dist</li>
    <li>python313Packages.python-aodhclient</li>
    <li>python313Packages.python-aodhclient.dist</li>
    <li>python313Packages.python-cinderclient</li>
    <li>python313Packages.python-cinderclient.dist</li>
    <li>python313Packages.python-designateclient</li>
    <li>python313Packages.python-designateclient.dist</li>
    <li>python313Packages.python-jenkins</li>
    <li>python313Packages.python-jenkins.dist</li>
    <li>python313Packages.python-keystoneclient</li>
    <li>python313Packages.python-keystoneclient.dist</li>
    <li>python313Packages.python-novaclient</li>
    <li>python313Packages.python-novaclient.dist</li>
    <li>python313Packages.python-octaviaclient</li>
    <li>python313Packages.python-octaviaclient.dist</li>
    <li>swiftclient (python313Packages.python-swiftclient)</li>
    <li>swiftclient.dist (python313Packages.python-swiftclient.dist)</li>
    <li>troveclient (python313Packages.python-troveclient)</li>
    <li>troveclient.dist (python313Packages.python-troveclient.dist)</li>
    <li>watcherclient (python313Packages.python-watcherclient)</li>
    <li>watcherclient.dist (python313Packages.python-watcherclient.dist)</li>
    <li>python313Packages.python-zaqarclient</li>
    <li>python313Packages.python-zaqarclient.dist</li>
    <li>zunclient (python313Packages.python-zunclient)</li>
    <li>zunclient.dist (python313Packages.python-zunclient.dist)</li>
    <li>python313Packages.stestr</li>
    <li>python313Packages.stestr.dist</li>
    <li>python313Packages.tempest</li>
    <li>python313Packages.tempest.dist</li>
    <li>python314Packages.cmd2</li>
    <li>python314Packages.cmd2-ext-test</li>
    <li>python314Packages.cmd2-ext-test.dist</li>
    <li>python314Packages.cmd2.dist</li>
    <li>python314Packages.pysim</li>
    <li>python314Packages.pysim.dist</li>
    <li>reno</li>
    <li>reno.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:x: 203 packages failed to build:</summary>
  <ul>
    <li>baboossh</li>
    <li>baboossh.dist</li>
    <li>barbicanclient (python313Packages.python-barbicanclient)</li>
    <li>barbicanclient.dist (python313Packages.python-barbicanclient.dist)</li>
    <li>bashate</li>
    <li>bashate.dist</li>
    <li>faraday-cli</li>
    <li>faraday-cli.dist</li>
    <li>glanceclient (python313Packages.python-glanceclient)</li>
    <li>glanceclient.dist (python313Packages.python-glanceclient.dist)</li>
    <li>heatclient (python313Packages.python-heatclient)</li>
    <li>heatclient.dist (python313Packages.python-heatclient.dist)</li>
    <li>html5validator</li>
    <li>html5validator.dist</li>
    <li>ironicclient (python313Packages.python-ironicclient)</li>
    <li>ironicclient.dist (python313Packages.python-ironicclient.dist)</li>
    <li>jenkins-job-builder (python313Packages.jenkins-job-builder)</li>
    <li>jenkins-job-builder.dist (python313Packages.jenkins-job-builder.dist)</li>
    <li>kolla</li>
    <li>kolla.dist</li>
    <li>magnumclient (python313Packages.python-magnumclient)</li>
    <li>magnumclient.dist (python313Packages.python-magnumclient.dist)</li>
    <li>manilaclient (python313Packages.python-manilaclient)</li>
    <li>manilaclient.dist (python313Packages.python-manilaclient.dist)</li>
    <li>mistralclient (python313Packages.python-mistralclient)</li>
    <li>mistralclient.dist (python313Packages.python-mistralclient.dist)</li>
    <li>neutronclient (python313Packages.python-neutronclient)</li>
    <li>neutronclient.dist (python313Packages.python-neutronclient.dist)</li>
    <li>nmap-parse</li>
    <li>openstackclient (python313Packages.python-openstackclient)</li>
    <li>openstackclient-full</li>
    <li>openstackclient-full.dist</li>
    <li>openstackclient.dist (python313Packages.python-openstackclient.dist)</li>
    <li>python313Packages.angrcli</li>
    <li>python313Packages.angrcli.dist</li>
    <li>python313Packages.cliff</li>
    <li>python313Packages.cliff.dist</li>
    <li>python313Packages.cmd2</li>
    <li>python313Packages.cmd2-ext-test</li>
    <li>python313Packages.cmd2-ext-test.dist</li>
    <li>python313Packages.cmd2.dist</li>
    <li>python313Packages.hacking</li>
    <li>python313Packages.hacking.dist</li>
    <li>python313Packages.keystoneauth1</li>
    <li>python313Packages.keystoneauth1.dist</li>
    <li>python313Packages.openstacksdk</li>
    <li>python313Packages.openstacksdk.dist</li>
    <li>python313Packages.openstacksdk.man</li>
    <li>python313Packages.os-client-config</li>
    <li>python313Packages.os-client-config.dist</li>
    <li>python313Packages.osc-lib</li>
    <li>python313Packages.osc-lib.dist</li>
    <li>python313Packages.osc-placement</li>
    <li>python313Packages.osc-placement.dist</li>
    <li>python313Packages.oslo-concurrency</li>
    <li>python313Packages.oslo-concurrency.dist</li>
    <li>python313Packages.oslo-config</li>
    <li>python313Packages.oslo-config.dist</li>
    <li>python313Packages.oslo-context</li>
    <li>python313Packages.oslo-context.dist</li>
    <li>python313Packages.oslo-db</li>
    <li>python313Packages.oslo-db.dist</li>
    <li>python313Packages.oslo-i18n</li>
    <li>python313Packages.oslo-i18n.dist</li>
    <li>python313Packages.oslo-log</li>
    <li>python313Packages.oslo-log.dist</li>
    <li>python313Packages.oslo-metrics</li>
    <li>python313Packages.oslo-metrics.dist</li>
    <li>python313Packages.oslo-serialization</li>
    <li>python313Packages.oslo-serialization.dist</li>
    <li>python313Packages.oslo-utils</li>
    <li>python313Packages.oslo-utils.dist</li>
    <li>python313Packages.osprofiler</li>
    <li>python313Packages.osprofiler.dist</li>
    <li>python313Packages.pyghmi</li>
    <li>python313Packages.pyghmi.dist</li>
    <li>python313Packages.pysim</li>
    <li>python313Packages.pysim.dist</li>
    <li>python313Packages.python-aodhclient</li>
    <li>python313Packages.python-aodhclient.dist</li>
    <li>python313Packages.python-cinderclient</li>
    <li>python313Packages.python-cinderclient.dist</li>
    <li>python313Packages.python-designateclient</li>
    <li>python313Packages.python-designateclient.dist</li>
    <li>python313Packages.python-jenkins</li>
    <li>python313Packages.python-jenkins.dist</li>
    <li>python313Packages.python-keystoneclient</li>
    <li>python313Packages.python-keystoneclient.dist</li>
    <li>python313Packages.python-novaclient</li>
    <li>python313Packages.python-novaclient.dist</li>
    <li>python313Packages.python-octaviaclient</li>
    <li>python313Packages.python-octaviaclient.dist</li>
    <li>swiftclient (python313Packages.python-swiftclient)</li>
    <li>swiftclient.dist (python313Packages.python-swiftclient.dist)</li>
    <li>troveclient (python313Packages.python-troveclient)</li>
    <li>troveclient.dist (python313Packages.python-troveclient.dist)</li>
    <li>watcherclient (python313Packages.python-watcherclient)</li>
    <li>watcherclient.dist (python313Packages.python-watcherclient.dist)</li>
    <li>python313Packages.python-zaqarclient</li>
    <li>python313Packages.python-zaqarclient.dist</li>
    <li>zunclient (python313Packages.python-zunclient)</li>
    <li>zunclient.dist (python313Packages.python-zunclient.dist)</li>
    <li>python313Packages.stestr</li>
    <li>python313Packages.stestr.dist</li>
    <li>python313Packages.tempest</li>
    <li>python313Packages.tempest.dist</li>
    <li>python314Packages.angrcli</li>
    <li>python314Packages.angrcli.dist</li>
    <li>python314Packages.cliff</li>
    <li>python314Packages.cliff.dist</li>
    <li>python314Packages.cmd2</li>
    <li>python314Packages.cmd2-ext-test</li>
    <li>python314Packages.cmd2-ext-test.dist</li>
    <li>python314Packages.cmd2.dist</li>
    <li>python314Packages.hacking</li>
    <li>python314Packages.hacking.dist</li>
    <li>python314Packages.jenkins-job-builder</li>
    <li>python314Packages.jenkins-job-builder.dist</li>
    <li>python314Packages.keystoneauth1</li>
    <li>python314Packages.keystoneauth1.dist</li>
    <li>python314Packages.openstacksdk</li>
    <li>python314Packages.openstacksdk.dist</li>
    <li>python314Packages.openstacksdk.man</li>
    <li>python314Packages.os-client-config</li>
    <li>python314Packages.os-client-config.dist</li>
    <li>python314Packages.osc-lib</li>
    <li>python314Packages.osc-lib.dist</li>
    <li>python314Packages.osc-placement</li>
    <li>python314Packages.osc-placement.dist</li>
    <li>python314Packages.oslo-concurrency</li>
    <li>python314Packages.oslo-concurrency.dist</li>
    <li>python314Packages.oslo-config</li>
    <li>python314Packages.oslo-config.dist</li>
    <li>python314Packages.oslo-context</li>
    <li>python314Packages.oslo-context.dist</li>
    <li>python314Packages.oslo-db</li>
    <li>python314Packages.oslo-db.dist</li>
    <li>python314Packages.oslo-i18n</li>
    <li>python314Packages.oslo-i18n.dist</li>
    <li>python314Packages.oslo-log</li>
    <li>python314Packages.oslo-log.dist</li>
    <li>python314Packages.oslo-metrics</li>
    <li>python314Packages.oslo-metrics.dist</li>
    <li>python314Packages.oslo-serialization</li>
    <li>python314Packages.oslo-serialization.dist</li>
    <li>python314Packages.oslo-utils</li>
    <li>python314Packages.oslo-utils.dist</li>
    <li>python314Packages.osprofiler</li>
    <li>python314Packages.osprofiler.dist</li>
    <li>python314Packages.pyghmi</li>
    <li>python314Packages.pyghmi.dist</li>
    <li>python314Packages.pysim</li>
    <li>python314Packages.pysim.dist</li>
    <li>python314Packages.python-aodhclient</li>
    <li>python314Packages.python-aodhclient.dist</li>
    <li>python314Packages.python-barbicanclient</li>
    <li>python314Packages.python-barbicanclient.dist</li>
    <li>python314Packages.python-cinderclient</li>
    <li>python314Packages.python-cinderclient.dist</li>
    <li>python314Packages.python-designateclient</li>
    <li>python314Packages.python-designateclient.dist</li>
    <li>python314Packages.python-glanceclient</li>
    <li>python314Packages.python-glanceclient.dist</li>
    <li>python314Packages.python-heatclient</li>
    <li>python314Packages.python-heatclient.dist</li>
    <li>python314Packages.python-ironicclient</li>
    <li>python314Packages.python-ironicclient.dist</li>
    <li>python314Packages.python-jenkins</li>
    <li>python314Packages.python-jenkins.dist</li>
    <li>python314Packages.python-keystoneclient</li>
    <li>python314Packages.python-keystoneclient.dist</li>
    <li>python314Packages.python-magnumclient</li>
    <li>python314Packages.python-magnumclient.dist</li>
    <li>python314Packages.python-manilaclient</li>
    <li>python314Packages.python-manilaclient.dist</li>
    <li>python314Packages.python-mistralclient</li>
    <li>python314Packages.python-mistralclient.dist</li>
    <li>python314Packages.python-neutronclient</li>
    <li>python314Packages.python-neutronclient.dist</li>
    <li>python314Packages.python-novaclient</li>
    <li>python314Packages.python-novaclient.dist</li>
    <li>python314Packages.python-octaviaclient</li>
    <li>python314Packages.python-octaviaclient.dist</li>
    <li>python314Packages.python-openstackclient</li>
    <li>python314Packages.python-openstackclient.dist</li>
    <li>python314Packages.python-swiftclient</li>
    <li>python314Packages.python-swiftclient.dist</li>
    <li>python314Packages.python-troveclient</li>
    <li>python314Packages.python-troveclient.dist</li>
    <li>python314Packages.python-watcherclient</li>
    <li>python314Packages.python-watcherclient.dist</li>
    <li>python314Packages.python-zaqarclient</li>
    <li>python314Packages.python-zaqarclient.dist</li>
    <li>python314Packages.python-zunclient</li>
    <li>python314Packages.python-zunclient.dist</li>
    <li>python314Packages.stestr</li>
    <li>python314Packages.stestr.dist</li>
    <li>python314Packages.tempest</li>
    <li>python314Packages.tempest.dist</li>
    <li>reno</li>
    <li>reno.dist</li>
    <li>sccmhunter</li>
    <li>sccmhunter.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>expliot</summary>
<pre>no configure script, doing nothing
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built expliot-0.11.1-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/expliot-0.11.1...OK
Repacking wheel as ./expliot-0.11.1-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;pythonRuntimeDepsCheckHook&quot; }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for expliot-0.11.1-py3-none-any.whl
  - cmd2&lt;3.0.0,&gt;=2.4.3 not satisfied by version 3.2.1</pre>
</details>
<details>
<summary>python314Packages.cliff</summary>
<pre>      File &quot;/nix/store/a43z433p9h0dywi2jp7pvhsmjxysqas4-python3.14-cliff-4.13.1/lib/python3.14/site-packages/cliff/sphinxext.py&quot;, line 451, in run
        return self._generate_app_node(app, application_name)
               ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
      File &quot;/nix/store/a43z433p9h0dywi2jp7pvhsmjxysqas4-python3.14-cliff-4.13.1/lib/python3.14/site-packages/cliff/sphinxext.py&quot;, line 363, in _generate_app_node
        for line in _format_parser(parser):
                    ~~~~~~~~~~~~~~^^^^^^^^
      File &quot;/nix/store/a43z433p9h0dywi2jp7pvhsmjxysqas4-python3.14-cliff-4.13.1/lib/python3.14/site-packages/cliff/sphinxext.py&quot;, line 228, in _format_parser
        for line in _format_usage(parser):
                    ~~~~~~~~~~~~~^^^^^^^^
      File &quot;/nix/store/a43z433p9h0dywi2jp7pvhsmjxysqas4-python3.14-cliff-4.13.1/lib/python3.14/site-packages/cliff/sphinxext.py&quot;, line 87, in _format_usage
        opt_usage = fmt._format_actions_usage(optionals, groups)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^
    AttributeError: &#x27;HelpFormatter&#x27; object has no attribute &#x27;_format_actions_usage&#x27;


The full traceback has been saved in:
/build/sphinx-err-vltf4ivx.log

To report this error to the developers, please open an issue at &lt;https://github.com/sphinx-doc/sphinx/issues/&gt;. Thanks!
Please also report this if it was a user error, so that a better error message can be provided next time.</pre>
</details>
<details>
<summary>sccmhunter</summary>
<pre>wrapping `/nix/store/6qb9a2q1s5piy4zaxjid2dan1fgnink1-sccmhunter-1.1.10/bin/sccmhunter.py&#x27;...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
Running phase: installCheckPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;installCheckPhase&quot; }
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;pythonCatchConflictsPhase&quot; }
Found duplicated packages in closure for dependency &#x27;rich&#x27;: 
  rich 14.3.3 (/nix/store/6n7vn5y82dgjvk8180xbjh261hdxc6ql-python3.12-rich-14.3.3)
    dependency chain:
      this derivation: /nix/store/6qb9a2q1s5piy4zaxjid2dan1fgnink1-sccmhunter-1.1.10
      ...depending on: /nix/store/7gkfrvpnyy62sa0cym6zddmddkszxfjr-python3.12-cmd2-3.2.1
      ...depending on: /nix/store/6n7vn5y82dgjvk8180xbjh261hdxc6ql-python3.12-rich-14.3.3
  rich 14.2.0 (/nix/store/7y4spvka0d6zlj6labyx9w791rjgihkj-python3.12-rich-14.2.0)
    dependency chain:
      this derivation: /nix/store/6qb9a2q1s5piy4zaxjid2dan1fgnink1-sccmhunter-1.1.10
      ...depending on: /nix/store/7y4spvka0d6zlj6labyx9w791rjgihkj-python3.12-rich-14.2.0

Package duplicates found in closure, see above. Usually this happens if two packages depend on different version of the same dependency.</pre>
</details>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>expliot</summary>
<pre>Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built expliot-0.11.1-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/expliot-0.11.1...OK
Repacking wheel as ./expliot-0.11.1-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for expliot-0.11.1-py3-none-any.whl
  - cmd2&lt;3.0.0,&gt;=2.4.3 not satisfied by version 3.2.1</pre>
</details>
<details>
<summary>python314Packages.cliff</summary>
<pre>      File &quot;/nix/store/f08bzv6q2hjqbyymaikr3h6fiaam2rzf-python3.14-cliff-4.13.1/lib/python3.14/site-packages/cliff/sphinxext.py&quot;, line 451, in run
        return self._generate_app_node(app, application_name)
               ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
      File &quot;/nix/store/f08bzv6q2hjqbyymaikr3h6fiaam2rzf-python3.14-cliff-4.13.1/lib/python3.14/site-packages/cliff/sphinxext.py&quot;, line 363, in _generate_app_node
        for line in _format_parser(parser):
                    ~~~~~~~~~~~~~~^^^^^^^^
      File &quot;/nix/store/f08bzv6q2hjqbyymaikr3h6fiaam2rzf-python3.14-cliff-4.13.1/lib/python3.14/site-packages/cliff/sphinxext.py&quot;, line 228, in _format_parser
        for line in _format_usage(parser):
                    ~~~~~~~~~~~~~^^^^^^^^
      File &quot;/nix/store/f08bzv6q2hjqbyymaikr3h6fiaam2rzf-python3.14-cliff-4.13.1/lib/python3.14/site-packages/cliff/sphinxext.py&quot;, line 87, in _format_usage
        opt_usage = fmt._format_actions_usage(optionals, groups)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^
    AttributeError: &#x27;HelpFormatter&#x27; object has no attribute &#x27;_format_actions_usage&#x27;


The full traceback has been saved in:
/build/sphinx-err-2ojphkfq.log

To report this error to the developers, please open an issue at &lt;https://github.com/sphinx-doc/sphinx/issues/&gt;. Thanks!
Please also report this if it was a user error, so that a better error message can be provided next time.</pre>
</details>
<details>
<summary>sccmhunter</summary>
<pre>patching script interpreter paths in /nix/store/zm2sdlpngzaldy4928gdq8fqjfs6jghb-sccmhunter-1.1.10-dist
Rewriting #!/nix/store/p3dgpm3z2s0k2sf6838ybrqvz6r89ry2-python3-3.12.12/bin/python3.12 to #!/nix/store/p3dgpm3z2s0k2sf6838ybrqvz6r89ry2-python3-3.12.12
wrapping `/nix/store/y0bhps2d5cnxwpahi5s9bil37zs66s7c-sccmhunter-1.1.10/bin/sccmhunter.py&#x27;...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
Running phase: installCheckPhase
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
Found duplicated packages in closure for dependency &#x27;rich&#x27;: 
  rich 14.3.3 (/nix/store/h4nqwybdlhqnc98k6zvh8aba4a573p6s-python3.12-rich-14.3.3)
    dependency chain:
      this derivation: /nix/store/y0bhps2d5cnxwpahi5s9bil37zs66s7c-sccmhunter-1.1.10
      ...depending on: /nix/store/qf5yvg3j49b07d68433lz25hysq629gc-python3.12-cmd2-3.2.1
      ...depending on: /nix/store/h4nqwybdlhqnc98k6zvh8aba4a573p6s-python3.12-rich-14.3.3
  rich 14.2.0 (/nix/store/78nzl62mm0fhb9aiyz19zpsh8mq5wgp7-python3.12-rich-14.2.0)
    dependency chain:
      this derivation: /nix/store/y0bhps2d5cnxwpahi5s9bil37zs66s7c-sccmhunter-1.1.10
      ...depending on: /nix/store/78nzl62mm0fhb9aiyz19zpsh8mq5wgp7-python3.12-rich-14.2.0

Package duplicates found in closure, see above. Usually this happens if two packages depend on different version of the same dependency.</pre>
</details>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python313Packages.cmd2</summary>
<pre>mocker = &lt;pytest_mock.plugin.MockerFixture object at 0x104969710&gt;
capsys = &lt;_pytest.capture.CaptureFixture object at 0x103c0a1d0&gt;

    def test_history_file_bad_json(mocker, capsys) -&gt; None:
        import lzma
    
        data = b&quot;THIS IS NOT JSON&quot;
        compressed_data = lzma.compress(data)
    
        history_file = &#x27;/tmp/doesntmatter&#x27;
&gt;       with open(history_file, &quot;wb&quot;) as f:
             ^^^^^^^^^^^^^^^^^^^^^^^^
E       PermissionError: [Errno 13] Permission denied: &#x27;/tmp/doesntmatter&#x27;

tests/test_history.py:943: PermissionError
=========================== short test summary info ============================
FAILED tests/test_completion.py::test_path_completion_complete_user - AssertionError: assert &#x27;~_nixbld11/&#x27; in []
FAILED tests/test_history.py::test_history_file_bad_compression - PermissionError: [Errno 13] Permission denied: &#x27;/tmp/doesntmatter&#x27;
FAILED tests/test_history.py::test_history_file_bad_json - PermissionError: [Errno 13] Permission denied: &#x27;/tmp/doesntmatter&#x27;
=========== 3 failed, 944 passed, 2 skipped, 35 deselected in 8.92s ============</pre>
</details>
<details>
<summary>python314Packages.cmd2</summary>
<pre>
    def test_path_completion_complete_user(cmd2_app) -&gt; None:
        import getpass
    
        user = getpass.getuser()
    
        text = f&#x27;~{user}&#x27;
        line = f&#x27;shell fake {text}&#x27;
        endidx = len(line)
        begidx = endidx - len(text)
        completions = cmd2_app.path_complete(text, line, begidx, endidx)
    
        expected = text + os.path.sep
&gt;       assert expected in completions
E       AssertionError: assert &#x27;~_nixbld12/&#x27; in []

tests/test_completion.py:650: AssertionError
=========================== short test summary info ============================
FAILED tests/test_completion.py::test_path_completion_complete_user - AssertionError: assert &#x27;~_nixbld12/&#x27; in []
=========== 1 failed, 946 passed, 2 skipped, 35 deselected in 8.55s ============</pre>
</details>
</details>

--- a/pkgs/development/python-modules/cmd2/default.nix
+++ b/pkgs/development/python-modules/cmd2/default.nix
@@ -12,16 +12,21 @@
   rich-argparse,
   setuptools-scm,
   wcwidth,
+  python,
 }:
 
 buildPythonPackage rec {
   pname = "cmd2";
-  version = "3.2.1";
+  version = if python.isPy313 then "3.1.0" else "3.2.1";
   pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-bGNyobJs0Uu2IJZTyJ1zAP58FDno3KMPW2tv/bXyFPo=";
+    hash =
+      if python.isPy313 then
+        "sha256-zOOuzgGLCxBVmIraoraHrJwd84v9Kr/Cnb61GpcH3jM="
+      else
+        "sha256-bGNyobJs0Uu2IJZTyJ1zAP58FDno3KMPW2tv/bXyFPo=";
   };
 
   build-system = [ setuptools-scm ];

--- a/pkgs/development/python-modules/rich-argparse/default.nix
+++ b/pkgs/development/python-modules/rich-argparse/default.nix
@@ -1,6 +1,7 @@
 {
   lib,
   buildPythonPackage,
+  python,
   fetchFromGitHub,
   hatchling,
   pytestCheckHook,
@@ -28,6 +29,8 @@ buildPythonPackage rec {
   disabledTests = [
     # coloring mismatch in fixture
     "test_subparsers_usage"
+  ]
+  ++ lib.optionals (!python.isPy313) [
     # solid vs dash line mismatch
     "test_rich_renderables"
   ];

--- a/pkgs/development/python-modules/rich/default.nix
+++ b/pkgs/development/python-modules/rich/default.nix
@@ -2,6 +2,7 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  python,
 
   # build-system
   poetry-core,
@@ -27,14 +28,18 @@
 
 buildPythonPackage rec {
   pname = "rich";
-  version = "14.3.3";
+  version = if python.isPy313 then "14.2.0" else "14.3.3";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "Textualize";
     repo = "rich";
     tag = "v${version}";
-    hash = "sha256-6udVO7N17ineQozlCG/tI9jJob811gqb4GtY50JZFb0=";
+    hash =
+      if python.isPy313 then
+        "sha256-oQbxRbZnVr/Ln+i/hpBw5FlpUp3gcp/7xsxi6onPkn8="
+      else
+        "sha256-6udVO7N17ineQozlCG/tI9jJob811gqb4GtY50JZFb0=";
   };
 
   build-system = [ poetry-core ];

